PyPDF2/_encryption.py

```diff
@@ -611,17 +625,21 @@ def __init__(
     ) -> None:
         # See TABLE 3.18 Entries common to all encryption dictionaries
         self.algV = algV
+        self.algR = algR
         self.entry = entry
         self.key_size = entry.get("/Length", 40)
         self.id1_entry = first_id_entry
         self.StmF = StmF
         self.StrF = StrF
         self.EFF = EFF
 
+        # 1 => owner password
+        # 2 => user password
+        self._password_type = 0
         self._key: Optional[bytes] = None
-        # keep key
-        self._user_keys: Dict = {}
-        self._owner_keys: Dict = {}
+
+    def verified(self) -> bool:
```

Comment:
I guess verified means that the file was decrypted? Would `is_decrypted` be better?

Reply:
Also, I think this should rather be `_verified` if the user does not need it.